gooddata-java/src/test/java/com/gooddata/sdk/service/project/ProjectServiceAT.java

```diff
@@ -150,6 +163,20 @@ public void getUserInProject() {
         assertThat(user, notNullValue());
     }
 
+    @Test(groups = {"project", "isolated_domain"}, dependsOnMethods = "disableUserInProject")
+    public void removeAccountFromProject() {
+        final User user = gd.getProjectService().getUser(project, account1);
+        assertThat(user, notNullValue());
+        gd.getProjectService().removeAccountFromProject(project, account1);
+
+        try {
+            gd.getProjectService().getUser(project, account1);
+            fail();
```

Comment:
you must use Hamcrest matchers for this, I will look for the solution as well

Reply:
well there's no other way (without including additional dependencies) than making something like this:
```java
assertThat("User was not deleted from the project", false)
```